package.json

```diff
@@ -134,7 +134,7 @@
     "resize-observer-polyfill": "^1.5.1",
     "screenfull": "^4.0.1",
     "semver": "^7.3.2",
-    "three": "github:mozillareality/three.js#hubs/master",
+    "three": "github:mozillareality/three.js#hubs/r128GLTFLoader",
```

Comment:
If https://github.com/MozillaReality/three.js/pull/57 is merged, I would update this line back to `master` in this PR or follow up PR. `package-lock.json` is same.

Reply:
I usually do it just before merging, and I leave a [ ] todo in the PR description to remind me